Cargo.toml

```diff
@@ -32,7 +32,7 @@ tokio = { version = "1.43.0", features = ["macros", "full"] }
 tokio-util = { version = "0.7.12", features = ["codec", "compat"] }
 tracing = "0.1.41"
 typed-builder = "0.20"
-winnow = "0.6.5"
+winnow = "0.6.26"
```

Comment:
0.6.26 deprecates (yes, the patchrelease) [ErrorKind](https://docs.rs/winnow/0.6.26/winnow/error/enum.ErrorKind.html) which breaks the build here.

"Lets do semantic versioning" they said. :angry: